Car_owners/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.10"
+
+services:
+  db:
+    build:
+      context: .
+      dockerfile: Dockerfile_postgres
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 15
+
+  django:
+    build:
+      context: .
+      dockerfile: Dockerfile_django
+    volumes:
+      - .:/Car_owners
+    ports:
+      - "8000:8000"
+    environment:
+      - POSTGRES_NAME=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    depends_on:
+      db:
+        condition: service_healthy
+    links:
+      - db
+    command: bash -c "
```

Comment:
that's fine but since it's a couple of lines of code already, better to extract it to separate shell script and run here either by cmd or entrypoint (see the difference here https://www.bmc.com/blogs/docker-cmd-vs-entrypoint/)